.github/workflows/goreleaser.yml

```diff
@@ -63,5 +61,5 @@ jobs:
       run: |
         make chart IMG=${{ env.REGISTRY }}/${{ env.OPERATOR_IMAGE }}:${{github.ref_name}}
```

Comment:
Note that this means that contents of the chart published do not correspond to any commit in git. That is a bit surprising and disconcerting.

Reply:
That content does correspond to a commit in git, in particular to the content for the tag used to make the release. You can verify this way:

```
$ helm pull oci://ghcr.io/kubestellar/ocm-status-addon-chart --version v0.2.0-alpha.1 -d /tmp --untar
$ grep -r image /tmp/ocm-status-addon-chart
/tmp/ocm-status-addon-chart/templates/operator.yaml:        image: ghcr.io/kubestellar/ocm-status-addon:0.2.0-alpha.1
/tmp/ocm-status-addon-chart/templates/operator.yaml:        imagePullPolicy: Always
```

Reply:
I think the `make chart ...` is in here because you want the chart to NOT EXACTLY match what is in git --- otherwise you could simply omit the `make chart`.

Reply:
correct, for example for the container image reference.